### PR TITLE
fix typos, clear up uni wording

### PR DESCRIPTION
**Pull request by t5r7**

### CODE REVIEW

File: README.md
Comment (t5r7) [inline README.md:4]: turns out I didn't actually fix the typo

File: README.md
Comment (mattiebw) [inline README.md]: careers people were mad that i didn't mention honours, should keep that in

**Review by mattiebw — CHANGES_REQUESTED** [review]

**Review by mattiebw — APPROVED** [review]

File: README.md
Comment (t5r7) [reply README.md]: used the full name of the course to please them, even linked to the page that says you're approved by tigers or something

**Comment by mattiebw** [comment]
LGTM, waiting for a review from my dad 👍

**Comment by t5r7** [comment]
> waiting for a review from my dad 👍

excuse me

**Comment by mattiebw** [comment]
Dad says its fine, merging

**Pull request merged**
